djinni/jni/djinni_support.cpp

```diff
@@ -0,0 +1,686 @@
+//
+// Copyright 2014 Dropbox, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "../djinni_common.hpp"
```

Comment:
I think it would be cooler to have `/djinni` set as an include dir, then you won't have to navigate upwards here. 🤔  

Why did you comment out the following in CMakeLists.txt?
```
target_include_directories(djinni_support_lib PUBLIC
 # $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/djinni>
```

Reply:
Ok I see you just copied that from the original code. 😛

Reply:
there is/might be  a relation to the generated files that I need to investigate
of course, I do not like the relative include either, but .... 

as said, this is just the first iteration step, not the final version
and yes, there are some TODOs, due to it's early nature

Reply:
Neat 👍 looking forward to getting rid of those nasty includes 😊